README.md

```diff
@@ -0,0 +1,87 @@
+# COSIMA Spack Configuration
+
+This repository contains the spack configuration and the spack environments used
+by COSIMA to deploy software on gadi.
+
+## Instalation instructions
+
+Clone this repository and its submodules to some appropriate location (e.g.,
+`/g/data/ik11/spack/0.20.1`):
+```bash
+$ git clone --recursive /g/data/ik11/spack/0.20.1
+```
+Next, create the python virtual environment:
+```bash
+$ cd /g/data/ik11/spack/0.20.1
+$ ./bootstrap_venv.sh
+```
+Finally, to use this spack installation one just needs to activate the python
+environment:
+```bash
+$  . /g/data/ik11/spack/0.20.1/venv/bin/activate
+$ which spack
+spack ()
+{ 
+    : this is a shell function from: /g/data/ik11/spack/0.20.1/spack/share/spack/setup-env.sh;
+    : the real spack script is here: /g/data/ik11/spack/0.20.1/spack/bin/spack;
+    _spack_shell_wrapper "$@";
+    return $?
+}
+```
+
+## Installing software
+
+It is recommended that all software be installed using spack
+environments. Currently two environments are provided (the names should be
+self-explanatory):
+1. `access-om3-devel`
+2. `common_tools_and_libraries`
+
+Installation of a spack environment is usually quite straightforward, but
+because this can be a CPU intensive operation and take quite some time, it is
+best to do this in parallel and to use an interactive job. 
+
+### Step-by-step instructions:
+
+ 1. Activate spack environment
+
+First activate the spack environment
+```bash
+$ spack env activate <env>
+```
+where `<env>` by the actual name of the environment.
+
+ 2. Download the sources
+
+As the compute nodes do not have internet access, one needs to download all the
```

Comment:
could use `copyq` queue to get internet access

Reply:
I think it's good practice to use spack mirrors: sources are stored in a central location and easily accessible.